skyreader/result.py

```diff
@@ -957,7 +918,7 @@ def bounding_box(ra, dec, theta, phi):
 
         print("preparing plot: {0}...".format(plot_filename))
 
-        cmap = matplotlib.cm.plasma_r
+        cmap = matplotlib.cm.plasma_r # mypy warning
```

Comment:
what's the issue here (and other places)?

Reply:
Something along the lines of `module has no attribute "plasma_r"`, although the code seems to work, still. It seems there is a more up to date, slightly different, API for this though https://matplotlib.org/stable/api/cm_api.html

I can try to replace it straight away.

Reply:
Replaced with `matplotlib.colormaps["plasma_r"]` and defined in a common attribute.